charts/kueue/values.yaml

```diff
@@ -0,0 +1,73 @@
+# Default values for kueue.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Enable each function, like kustomize https://github.com/kubernetes-sigs/kueue/blob/main/config/default/kustomization.yaml
+enablePrometheus: false
+enableCertManager: false
+enableWebhook: true
+
+# Customize controlerManager
+controllerManager:
+  kubeRbacProxy:
+    image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+  manager:
+    image: gcr.io/k8s-staging-kueue/kueue:main
```

Comment:
Is not recommended to have the full image name like this, is better as the comment above, to have more control over repository and tag. 
Also pullPolicy should be `IfNotPresent` on release cuts and `Always` on master branch (best practice)
also having imagePullSecrets in case is an AirGapped Cluster

Reply:
Thank you for your comments.

@ArangoGutierrez , @alculquicondor , @moficodes 
BTW, which image should I use here?
Should I use "gcr.io/k8s-staging-kueue/kueue:v0.3.0" if I release this PR for kueue@v0.3.0 ?